src/properties.ts

```diff
@@ -246,8 +249,8 @@ function getSonarFileProperties(projectBaseDir: string): ScannerProperties {
 
     return properties;
   } catch (error) {
-    log(LogLevel.WARN, `Failed to read ${SONAR_PROJECT_FILENAME} file: ${error}`);
-    throw error;
+    log(LogLevel.DEBUG, `Failed to read ${SONAR_PROJECT_FILENAME} file: ${error}`);
```

Comment:
Changed this to `DEBUG` because it's a valid use-case not to have a `sonar-project.propertie` file present